translate.mjs

```diff
@@ -0,0 +1,37 @@
+import fetch from 'node-fetch';
```

Comment:
@stajs IMO the Michael Jackson Script imports are nicer and we should swap out those common JS imports in the other scripts.